### PR TITLE
update start-notebook.sh and k8s-start-notebook.sh

### PR DESCRIPTION
**Pull request by zzti-bsj**
1. add `k8s-start-notebook.sh`
2. update deployment.yaml to support run `start-notebook.sh`

### CODE REVIEW

File: docker/bigdl/Dockerfile [hunk 1]
```diff
@@ -103,6 +104,7 @@ ENV RUNTIME_DRIVER_CORES            ${RUNTIME_DRIVER_CORES}
 ENV RUNTIME_DRIVER_MEMORY           ${RUNTIME_DRIVER_MEMORY}
 ENV RUNTIME_EXECUTOR_CORES          ${RUNTIME_EXECUTOR_CORES}
 ENV RUNTIME_EXECUTOR_MEMORY         ${RUNTIME_EXECUTOR_MEMORY}
+ENV RUNTIME_EXECUTOR_INSTANCES      ${RUNTIME_EXECUTOR_INSTANCES}
 ENV RUNTIME_TOTAL_EXECUTOR_CORES    ${RUNTIME_TOTAL_EXECUTOR_CORES}
```
Comment (glorysdj) [inline docker/bigdl/Dockerfile:108]: also delete this RUNTIME_TOTAL_EXECUTOR_CORES    
we use RUNTIME_EXECUTOR_INSTANCES to replace RUNTIME_TOTAL_EXECUTOR_CORES    
and please update the doc

Comment (zzti-bsj) [reply docker/bigdl/Dockerfile]: Completed, please take a look.

Comment (zzti-bsj) [reply docker/bigdl/Dockerfile]: Completed, please take a look.

**Review by glorysdj — APPROVED** [review]
LGTM

**Pull request merged**